rules/docker-base.mk

```diff
@@ -2,6 +2,7 @@
 
 DOCKER_BASE = docker-base.gz
 $(DOCKER_BASE)_PATH = $(DOCKERS_PATH)/docker-base
+$(DOCKER_BASE)_DEPENDS += $(SUPERVISOR)
```

Comment:
>SUPERVISOR [](start = 28, length = 10)

Suggest not include built packages into docker-base.
All component here are directly downloaded from debian sources server, and main updated for security update.
supervisor is likely to update more often than them, like a development package.